deploy/olm-catalog/ibm-common-service-operator/3.4.1/ibm-common-service-operator.v3.4.1.clusterserviceversion.yaml

```diff
@@ -531,8 +531,26 @@ metadata:
         - secretname: ibmcloud-cluster-ca-cert
           sharewith:
           - namespace: kube-public
+        - secretname: icp-serviceid-apikey-secret
+          sharewith:
+          - namespace: kube-system
+        - secretname: platform-oidc-credentials
+          sharewith:
+          - namespace: kube-system
+        - secretname: icp-mongodb-admin
+          sharewith:
+          - namespace: kube-system
+        - secretname: icp-mongodb-client-cert
+          sharewith:
+          - namespace: kube-system
+        - secretname: cs-ca-certificate-secret
```

Comment:
This secret does not exist in 3.2.4

Reply:
@chenzhiwei This is in place of `cluster-ca-cert` that was used by CS+MCM in previous versions. This is the one that is now available to communicate correctly with CS as was done previously.